src/components/FiltersDrawer/FiltersDrawer.test.tsx

```diff
@@ -2,16 +2,9 @@ import { generateSnapshots } from '@chanzuckerberg/story-utils';
 import { composeStories } from '@storybook/testing-react';
 import { fireEvent, render, screen } from '@testing-library/react';
 import React from 'react';
-import * as stories from './Filters.stories';
+import * as stories from './FiltersDrawer.stories';
 
 const { OverflowInteractive } = composeStories(stories);
-const consoleLogMock = jest.spyOn(console, 'log').mockImplementation();
-beforeEach(() => {
-  consoleLogMock.mockClear();
-});
-afterAll(() => {
-  consoleLogMock.mockRestore();
-});
```

Comment:
Was this just not being used?

Reply:
I think there used to be `console logs` in the code but it was no longer happening so I removed it